docs/api/v3.rst

```diff
@@ -863,9 +917,63 @@ Translations listing
             "count": 25,
             "next": "/api/v3/projects/pip/translations/?limit=10&offset=10",
             "previous": null,
-            "results": ["PROJECT"]
+            "results": [{
+                "id": 12345,
+                "name": "Pip",
+                "slug": "pip",
+                "created": "2010-10-23T18:12:31+00:00",
+                "modified": "2018-12-11T07:21:11+00:00",
+                "language": {
+                    "code": "en",
+                    "name": "English"
+                },
+                "programming_language": {
+                    "code": "py",
+                    "name": "Python"
+                },
+                "repository": {
+                    "url": "https://github.com/pypa/pip",
+                    "type": "git"
+                },
+                "default_version": "stable",
+                "default_branch": "master",
+                "subproject_of": null,
+                "translation_of": null,
+                "urls": {
+                    "documentation": "http://pip.pypa.io/en/stable/",
+                    "home": "https://pip.pypa.io/"
+                },
+                "tags": [
+                    "disutils",
+                    "easy_install",
+                    "egg",
+                    "setuptools",
+                    "virtualenv"
+                ],
+                "users": [
+                    {
+                        "username": "dstufft"
+                    }
+                ],
+                "active_versions": {
+                    "stable": "{VERSION}",
+                    "latest": "{VERSION}",
+                    "19.0.2": "{VERSION}"
+                },
+                "_links": {
+                    "_self": "/api/v3/projects/pip/",
+                    "versions": "/api/v3/projects/pip/versions/",
+                    "builds": "/api/v3/projects/pip/builds/",
+                    "subprojects": "/api/v3/projects/pip/subprojects/",
+                    "superproject": "/api/v3/projects/pip/superproject/",
+                    "redirects": "/api/v3/projects/pip/redirects/",
+                    "translations": "/api/v3/projects/pip/translations/"
+                }
+            }]
         }
 
+        The ``results`` in response is an array of project data, which is same to the `Project API <https://docs.readthedocs.io/en/stable/api/v3.html#get--api-v3-projects-(string-project_slug)->`_.
+
```

Comment:
The link should be a ref role, not a hardcoded url

Similar to `` :http:get:`/api/v2/project/(int:id)/` ``

Reply:
I'm not so sure about this `http` directive, and cannot find existing usage by searching `<http:[^/]` in project. Let me push untested code to check the behavior.